sketch.js

```diff
@@ -82,6 +87,7 @@ function gameover() {
   text("HIT", width / 2, height / 2);
   maxScore = max(score, maxScore);
   score = 0;
+  gameisover = true;
```

Comment:
wrong place? showScores is for showing scores. There is gameover function :D

And, may be it's better to do some dying animation or just leave it as it is?

Becase if you do machinelearning you can just tell ML that you lost, change your genoma, no need to restart the game.

However, it can loose at the beginning all the time, bacause tubes start at different position... So yeah, it should be changed in some way, but not with noLoop() :((